sound/soc/sof/intel/hda-dsp.c

```diff
@@ -282,7 +282,7 @@ void hda_dsp_ipc_int_disable(struct snd_sof_dev *sdev)
 			HDA_DSP_REG_HIPCCTL_BUSY | HDA_DSP_REG_HIPCCTL_DONE, 0);
 }
 
-static int hda_suspend(struct snd_sof_dev *sdev, int state)
+static int hda_suspend(struct snd_sof_dev *sdev, int state, bool resume_runtime)
```

Comment:
should be suspend_runtime...

Reply:
very sorry, Thanks Pierre. I will correct it.